src/XIVLauncher.Core/Program.cs

```diff
@@ -44,7 +44,9 @@ class Program
 
     private static readonly Vector3 clearColor = new(0.1f, 0.1f, 0.1f);
     private static bool showImGuiDemoWindow = true;
-
+    public static string Distro { get; private set; }
```

Comment:
Might be an idea to make this an enum so we're more aware of the valid values for it, could then just assign an attribute of its url value so we cna download the wine version from it using that attribute name.